test/cts/from_nnapi/README.md

```diff
@@ -0,0 +1,99 @@
+# Converted CTS from NNAPI CTS
+
+## Test Generator Tool
+This [test_generator](./test_generator) tool is for converting existed native 
+[NNAPI CTS](https://android.googlesource.com/platform/frameworks/ml/+/refs/tags/android-cts-10.0_r5/nn/runtime/test/specs/) of V1_0, V1_1 and V1_2 versions to these tests for WebNN API.
+
+### NNAPI Operations being map to WebNN API Operations Tables
+* Part I 
+
+The `test_generator` tool has supported to convert those tests of following
+NNAPI operations to the tests for such WebNN API operations of first wave.
+
+| NNAPI                         | WebNN API (first wave ops)    |
+|:------------------------------|:------------------------------|
+| RELU1                         | clamp                         |
+| RELU6                         | clamp                         |
+| CONCATENATION                 | concat                        |
+| ADD                           | add [+ relu/clamp]            |
+| SUB                           | sub [+ relu/clamp]            |
+| MUL                           | mul [+ relu/clamp]            |
+| DIV                           | div [+ relu/clamp]            |
+| MAXIMUM                       | max                           |
+| MINIMUM                       | min                           |
+| EXP                           | exp                           |
+| LOGISTIC                      | sigmoid                       |
+| SQRT                          | sqrt                          |
+| TANH                          | tanh                          |
+| AVERAGE_POOL_2D               | averagePool2d [+ relu/clamp]  |
+| MAX_POOL_2D                   | maxPool2d  [+ relu/clamp]     |
+| CONV_2D                       | conv2d [+ add + relu/clamp]   |
+| DEPTHWISE_CONV_2D             | conv2d [+ add + relu/clamp]   |
+| GROUPED_CONV_2D               | conv2d [+ add + relu/clamp]   |
+| RELU                          | relu                          |
+| RESHAPE                       | reshape                       |
+| SLICE                         | slice                         |
+| SOFTMAX                       | softmax + exp                 |
```

Comment:
Why does `SOFTMAX` map to `softmax` + `exp`?

Reply:
I didn't find the test cases for softmax, did I miss anything?

Reply:
> Why does `SOFTMAX` map to `softmax` + `exp`?
@huningxin Sorry to give this wrong mapping message, actually SOFTMAX` maps to `softmax`.  The converting NNAPI 'SOFTMAX' to WebNN 'softmax' is ongoing, the commit will be submitted later.

Reply:
@huningxin Now I've add 6 (passed 6) softmax test, please take a look, thanks.